doc/nrf/libraries/caf/ble_bond.rst

```diff
@@ -0,0 +1,62 @@
+.. _caf_ble_bond:
+
+CAF: Bluetooth LE bond module
+#############################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The |ble_bond| of the :ref:`lib_caf` (CAF) allows to introduce Bluetooth LE bonding to the application.
+The module also allows to delete bonds for the default Bluetooth local identity.
+
+Configuration
+*************
+
+Before using the module make sure that the following Kconfig options are enabled:
+
+* :kconfig:`CONFIG_BT_BONDABLE`
+* :kconfig:`CONFIG_BT_SETTINGS`
+* :kconfig:`CONFIG_CAF_SETTINGS_LOADER`
+* :kconfig:`CONFIG_CAF_BLE_COMMON_EVENTS`
+
+The device must be Bluetooth bondable and bonds must be stored in non-volatile memory (settings).
+The :ref:`caf_settings_loader` is used to load content of the settings area during boot.
+
+Enable the |ble_bond| module using :kconfig:`CONFIG_CAF_BLE_BOND`.
```

Comment:
```suggestion
The |ble_bond| module is enabled using :kconfig:`CONFIG_CAF_BLE_BOND`.
```

Reply:
Should we use active or passive form here? @greg-fer preferred active form if I recall correctly

Reply:
Yes, we should be using active tense, but when we start a new paragraph with a new sentence, it’s good to start with a noun.

Reply:
Fixed